DataManagement/GitTutorial.Rmd

```diff
@@ -196,60 +218,100 @@ git clone git:kingsleyshacklebolt/dragon_study.git
 
 
 
 ## GitHub workflow
 
-If you create or edit a file in your repository and save the chages the file will turn up in the git panel labelled with two yellow question marks.
+### Commit
+
+If you create or edit a file in your repository and save the changes the file will appear in the **Git** panel.
+There will be two yellow boxes with **question marks** if you add a file, a blue box with a **M** if you edit a file that has already been committed.
+And a red box with a **D** if you delete a file. 
+
+<br/><br/>
+```{r changed_file}
+knitr::include_graphics("Pics/gittutorial/18_changed_files.png")
+```
+<br/><br/>
+
+Once you have written a chunk of code, save it and click on the **Commit** button. 
+A new window will appear.
 
-!!! Picture of git window !!!
+<br/><br/>
+```{r commit_window}
+knitr::include_graphics("Pics/gittutorial/19_commit_window.png")
+```
+<br/><br/>
+
+All the changes in the file are shown in green and red color.
+Green is code that you have added to the file.
+Red is code that has been deleted.
+This makes it very easy to see all the changes that haven been made.
+
+Stage the changes you made to the **Commit message** (top right). 
+The commit message should contain all the changes you have done.
+It can be short, but should be complete.
+It will help you later if you are searching for a specific commit.
 
-Once you have written a chunk of code, save it and click on the commit button. 
-A new window will appear. 
-Stage the changes you made to the commit message (top right). 
-Write a useful message, it will help you later if you are searching for a specific commit. 
-Click commit to commit these changes which creates a permanent snapshot of the file in the Git directory along with a message that indicates what you did to the file. 
-So far you are still working locally on your computer and you have not done anything to the remote repository on GitHub.
+Note that you need to tick all files that you want to commit.
 
-!!! picture of commit window !!!
 
-All the changes in the file will be shown in green and red color.
+Click **Commit** to save the changes which creates a permanent snapshot of the file in the Git directory along with a message that describe the changes you made in this file. 
 
-Green: code you have added
 
-Red: code you have deleted
+### Some rules
 
-!!! picture of green and red lines !!!
+Commits are cheap.
+It does not take much time to click on **Commit**, write a few words or sentences about the commit and stage it.
+Therefore, commit often and provide useful messages so you can keep track of what you are doing. 
 
-Commits are cheap. 
-Commit often and provide useful messages so you can keep track of what you are doing. 
-Don’t do this:
+This is a bad example of a workflow:
 <br/><br/>
 ```{r git_commit}
-knitr::include_graphics("Pics/gittutorial/17_git_commit.png")
+knitr::include_graphics("Pics/gittutorial/20_git_commit.png")
 ```
 
-(From xkcd)
+(Picture from xkcd)
```

Comment:
Not really a bad workflow, just bad commit messages. Bad workflow would be "all changes since last year". 
add link - not least so readers can see the alt-text

Reply:
What link do you mean here?